File: .sdk/tm/csharp/test/ExistsTest.cs
// BluefinShieldconexMgmt SDK exists test.

using Xunit;

using BluefinShieldconexMgmtSdk;

namespace BluefinShieldconexMgmtSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinShieldconexMgmtSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}

File: .sdk/tm/js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { BluefinShieldconexMgmtSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await BluefinShieldconexMgmtSDK.test()
    equal(null !== testsdk, true)
  })

})
